code/src/utest/pcgen/testsupport/StrangeMap.java

```diff
@@ -21,7 +21,7 @@
 
 public class StrangeMap<K, V> extends HashMap<K, V>
 {
-	public StrangeMap(String s)
+	public StrangeMap()
```

Comment:
Tests may also depend on this being "strange" - need to check before this is modified.  Some of these intentionally do weird stuff to break assumptions made by certain reflection calls